### PR TITLE
Fix how UUID checked

### PR DESCRIPTION
**Pull request by squirrelo**
This fixes a bug where some valid UUIDs were being flagged as invalid.

### CODE REVIEW

**Comment by mortonjt** [comment]
I think this looks good.

**Comment by squirrelo** [comment]
Can this get a quick review/merge by someone else? @wasade @ElDeveloper @josenavas

File: amgut/lib/data_access/ag_data_access.py [hunk 1]
```diff
@@ -231,9 +231,8 @@ def registerHandoutKit(self, ag_login_id, supplied_kit_id):
             Non-UUID4 value sent as ag_login_id
         """
         with TRN:
-            uuid_check = UUID(ag_login_id)
-            if not uuid_check.version == 4:
-                raise ValueError("Non-UUID4 value passed: %s" % ag_login_id)
+            # make sure properly formatted UUID passed in
+            UUID(ag_login_id, version=4)
```
Comment (ElDeveloper) [inline amgut/lib/data_access/ag_data_access.py]: Can you fix the raises section? It no longer raises a ValueError, it raises a UUID exception, right?

Comment (squirrelo) [inline amgut/lib/data_access/ag_data_access.py]: The UUID module raises a ValueError:
```python
>>> from uuid import UUID
>>> UUID('bad', version=4)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/uuid.py", line 134, in __init__
    raise ValueError('badly formed hexadecimal UUID string')
ValueError: badly formed hexadecimal UUID string
```

Comment (ElDeveloper) [inline amgut/lib/data_access/ag_data_access.py]: My bad, I thought I saw something different last night, thanks for checking!

**Pull request merged**